parmoo/optimizers/gps_search.py

```diff
@@ -150,26 +151,24 @@ def solve(self, x):
             raise TypeError("x must be a numpy array")
         # Initialize an empty list of results
         result = []
+        lb_tmp = np.zeros(self.n)
+        ub_tmp = np.ones(self.n)
         # For each acqusisition function
         for j, acquisition in enumerate(self.acquisitions):
             # Create a new trust region
             rad = self.resetObjectives(x[j, :])
-            lb_tmp = np.zeros(self.n)
-            ub_tmp = np.ones(self.n)
-            for i in range(self.n):
-                lb_tmp[i] = max(self.lb[i], x[j, i] - rad)
-                ub_tmp[i] = min(self.ub[i], x[j, i] + rad)
+            lb_tmp[:] = np.maximum(self.lb[:], x[j, :] - rad)
```

Comment:
This is interesting, probably should investigate all max and min occurrences throughout the codebase.

Reply:
Yeah, I adjust here to allow for the trust region to be scaled with the design tolerance, since I may allow for it to decay further in certain variables (with low ``des_tol`` values) than others